kubernetes/fineract-server-deployment.yml

```diff
@@ -61,16 +61,23 @@ spec:
       containers:
       - name: fineract-server
         image: apache/fineract:latest
+        resources:
+          limits:
+            cpu: "1000m"
+            memory: "1Gi"
```

Comment:
One MAY want to give it more, of course. I'm not sure what we should put as a default. Personally I would have probably given it a bit more, actually. But having one, any, is better than none at all, to start with; we can always tweak it.